clang/test/Preprocessor/sycl-esimd-force-stateless-mem.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang_cc1 %s -fsycl-is-device -fsycl-esimd-force-stateless-mem -E -dM | FileCheck --check-prefix=CHECK-OPT %s
+
```

Comment:
Please add a comment describing test

Reply:
Added. Thank you. fc687f617ef5a730bcd9436e4d6cdd5ea33354ef